app/models/solr_document.rb

```diff
@@ -46,6 +26,24 @@ class SolrDocument
   # Use field names from Dublin Core
   use_extension(Blacklight::Document::DublinCore)
 
+  # Blacklight OAI metadata mapping
+  field_semantics.merge!(
+    title:       "title_tesim",
+    description: "description_tesim",
+    langauge: "language_label_ssm",
+    subject: "lc_subject_label_tesim",
+    contributor: "all_contributors_label_sim",
```

Comment:
Is this the hidden contributors field?

Reply:
I'm not sure if it is hidden in the display

Reply:
@chrissyrissmeyer yes https://github.com/ucsblibrary/alexandria/blob/228c87c9b3e2df9f5e3673f1913caa9845967d0b/app/indexers/object_indexer.rb#L45-L46

Reply:
Great! The only potential issue here is that "creator_tesim" values also show up here, so we'd end up with duplicates for all records that have names whose role is identified as "creator" -- but we currently think that is preferable to not having creators show up at all.